webknossos/tests/conftest.py

```diff
@@ -318,10 +317,7 @@ def pytest_collection_modifyitems(items: List[pytest.Item]) -> None:
         if item.get_closest_marker("vcr") is None:
             item.add_marker("vcr")
 
-        if (
-            item.get_closest_marker("block_network") is None
-            and multiprocessing.get_start_method() != "fork"
-        ):
+        if item.get_closest_marker("block_network") is None:
             # To allow for UNIX socket communication necessary for spawn multiprocessing
             # addresses starting with `/` are allowed
             marker = pytest.mark.block_network(allowed_hosts=["/.*"])
```

Comment:
I think you might also add `"/.*" to the list of allowed hosts, if the marker exist already, e.g.
```python
marker = item.get_closest_marker("block_network")
if marker is None:
    marker = pytest.mark.block_network(allowed_hosts=["/.*"])
    item.add_marker(marker)
else:
    marker.kwargs["allowed_hosts"].append("/.*")
```

Then you can omit the additional `"/.*"` directly at the tests.